.github/workflows/validation_workflow.yml

```diff
@@ -13,14 +13,21 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: [ "3.10", "3.8","3.9"]
```

Comment:
A quick heads-up on the different python versions, do we have a timeline for how long we want to support each version? Or if we always want to test all three currently? Can this be simplified after figuring out which sites are used by those running the workflows actually (like, with a survey) such that one does not need to provide backwards-compatibility to e.g. 3.8? I guess (from our other discussions) 3.9 seems to be the go-to version which also runs at lxplus.
This is not a strict requirement for this PR, but rather a comment (as it's listed here and we always try to optimize things, I thought I'd bring this up).

Reply:
It would be best to stick with one python version indeed. Concerning lxplus: the BTVNanoCommissioning setup runs there successfully with python 3.10. I do not have a strong opinion on whether to support 3.9 or 3.10, python 3.8 might become too old soon though. We should also make sure that the installation readme mentions the correct python version for the setup

Reply:
Absolutely, we could also just focus on two of the three versions (3.9/3.10), save 1/3 of the tests (time) and related to that save person power to make the setup compatible with 3.8, which could be spend on more detailed studies with the state-of-the-art versions.

Reply:
@Ming-Yan please restrict the support to python 3.9 and 3.10 only as discussed above

Reply:
and adjust the README accordingly (for setting up the conda environment)

Reply:
Do you have specific issues with py38 compatibility? Dropping support for versions before they reach EOL is rather radical and should have a good justification. Otherwise, if it's necessary to save core hours on actions dropping the intermediate versions is better because it's a fair assumption the code will work if it works on both the previous and following versions.

Reply:
Hi @andrzejnovak! We do not have issues with python 3.8, but the idea was to only support 3.9 and 3.10 such that the amount of CI pipeline tests is reduced. I am fine with dropping 3.9 instead

Reply:
I am not sure I understand the issue with multiple tests unless we do actually manage to hit some GHA limit. They run concurrently no?

Reply:
they do run in parallel, but sometimes, one of the 3 versions takes longer, which might be a hint for some GHA limit being reached?